src/plugins/saved_objects_management/public/management_section/objects_table/components/duplicate_result_flyout.tsx

```diff
@@ -91,7 +91,7 @@ export class DuplicateResultFlyout extends React.Component<DuplicateResultFlyout
         {copiedCount && (
           <EuiFlexItem grow={false}>
             <EuiTitle size="xs">
-              <h4 className="savedObjectsManagementImportSummary__copiedCountCount">
+              <h4 className="savedObjectsManagementImportSummary__createdCount">
```

Comment:
Nit: This change does not seem related to the copy issue, was it made intentionally?